core/variant/variant_call.cpp

```diff
@@ -1928,18 +1930,28 @@ static void _register_variant_builtin_methods() {
 	bind_method(Basis, inverse, sarray(), varray());
 	bind_method(Basis, transposed, sarray(), varray());
 	bind_method(Basis, orthonormalized, sarray(), varray());
+	bind_method(Basis, orthogonalized, sarray(), varray());
 	bind_method(Basis, determinant, sarray(), varray());
 	bind_methodv(Basis, rotated, static_cast<Basis (Basis::*)(const Vector3 &, real_t) const>(&Basis::rotated), sarray("axis", "angle"), varray());
 	bind_method(Basis, scaled, sarray("scale"), varray());
+	bind_method(Basis, scaled_local, sarray("scale"), varray());
+	bind_method(Basis, scaled_orthogonal, sarray("scale"), varray());
 	bind_method(Basis, get_scale, sarray(), varray());
+	bind_method(Basis, get_scale_local, sarray(), varray());
+	bind_method(Basis, get_scale_abs, sarray(), varray());
 	bind_method(Basis, get_euler, sarray("order"), varray((int64_t)EulerOrder::YXZ));
+	bind_method(Basis, set_euler, sarray("euler", "order"), varray());
```

Comment:
`Quaternion` also has a `get_euler` method but not `set_euler`, should it also be added there?

Reply:
Since the quaternion and euler conversion results cannot be determined uniquely, so I think it needs to be implemented as a wrapper function for Basis, is this necessary?

Reply:
Well, the `Quaternion::from_euler()` cannot specify the order in the first place, so we need to improve it from there. Let's leave that for later.